tests/conftest.py

```diff
@@ -33,17 +33,19 @@ def settings():
 
 
 @pytest.fixture(autouse=True)
-def mocked_bugzilla(request):
+def mocked_bugzilla(request, monkeypatch):
     if "no_mocked_bugzilla" in request.keywords:
         yield None
     else:
-        with mock.patch("jbi.services.BugzillaClient") as mocked_bz:
+        monkeypatch.setattr("jbi.services.bugzilla._CLIENT", None)
```

Comment:
You didn't use lru_cache because you needed a way to mock? 
I think you could call `cache_clear()` on a lru_cache decorated function

Reply:
Good call, thanks!